aqua/qt6/Portfile

```diff
@@ -886,14 +906,28 @@ foreach {module module_info} [array get modules] {
             configure.dir                ${workpath}/build
 
             # Qt suggests using Ninja to build
+            depends_build-append         port:ninja
```

Comment:
Note: It is merely my personal preference to specify this dependency in one place, rather than repeatedly in the `modules` array; to me it is clear that the dependencies in the `modules` array are not necessarily exhaustive. But if others prefer/find it more clear to specify it for each module, I can do that instead.